Tests/tx_test.py

```diff
@@ -196,3 +196,13 @@ def test_long_charstring_bug444():
     actual_path = runner(CMD + ['-o', '0', '-f', 'CJK-VarTest.otf'])
     expected_path = _get_expected_path('CJK-VarTest.txt')
     assert differ([expected_path, actual_path, '-s', '## Filename'])
+
+def test_many_hints_string_bug354():
+    # The glyph T@gid002 has 33 hstem hints. This tests a bug where
+    # tx defined an array of only 6 operants.
+    # This is encountered only when wrinting to a VF CFF2.
+    actual_path = runner(CMD + ['-o', 'cff2', '-f', 'cff2_vf.otf'])
+    actual_path = runner(CMD + ['-f', actual_path, '-o', 'dcf'])
```

Comment:
I think you need to use `-a` option in this second call to `runner`

Reply:
Don't use `actual_path` twice, because in case of failure it may be less clear which one is